pkg/health/pods.go

```diff
@@ -87,15 +90,34 @@ func PodManagersRecreated(client k8sclient.Client, options *k8sclient.ListOption
 }
 
 // DaemonSetsRecreated checks the number of pod replicas managed by DaemonSets
-func DaemonSetsRecreated(client k8sclient.Client, options *k8sclient.ListOptions) (bool, error) {
-	daemonSetList := v1beta1.DaemonSetList{}
-
-	err := client.List(context.TODO(), options, &daemonSetList)
+func DaemonSetsRecreated(
```

Comment:
@Danil-Grigorev same question here: is this necessary if we use @jortel change that auto-downgrades versions when necessary?

Reply:
It is unrelated to the PR